styles/components/_slate.scss

```diff
@@ -9,10 +9,14 @@
 	&.-plain { background: $brand-white; }
 
   // For consistency
-  h1, h2, h3, h4, h5, h6, p {
+  h1, h2, h3, h4, h5, h6 {
     margin-bottom: $spacer-large;
   }
 
+  p {
+    margin-bottom: $spacer-small;
```

Comment:
But this means you have small space between the last paragraph and the button, right?